app/views/users/show.json.jbuilder

```diff
@@ -0,0 +1 @@
+json.partial! "users/user", user: @user
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.